commands/hush.go

```diff
@@ -0,0 +1,87 @@
+package commands
+
+import (
+	"fmt"
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api/v5"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var baseHushDir = filepath.Join(os.TempDir(), "/bendan/hush")
+
+func init() {
+	err := os.MkdirAll(baseHushDir, 0755)
+	if err != nil {
+		log.Println("hush init failed: ", err)
+		return
+	}
+}
+
+func readTimeFromFile(chatID string) (time.Time, error) {
+	filePath := filepath.Join(filepath.Join(baseHushDir, chatID))
+	fileContent, err := os.ReadFile(filePath)
+	if err != nil {
+		return time.Time{}, fmt.Errorf("readTimeFromFile: %v", err)
+	}
+
+	unixTime, err := strconv.ParseInt(string(fileContent), 10, 64)
+	if err != nil {
+		return time.Time{}, fmt.Errorf("readTimeFromFile: %v", err)
+	}
+
+	timestamp := time.Unix(unixTime, 0)
+	return timestamp, nil
+}
+
+func writeTimeToFile(chatID string) error {
+	filePath := filepath.Join(filepath.Join(baseHushDir, chatID))
+	after30Minutes := time.Now().Add(30 * time.Minute)
+	fileContent := strconv.Itoa(int(after30Minutes.Unix()))
+	err := os.WriteFile(filePath, []byte(fileContent), 0644)
+
+	if err != nil {
+		return fmt.Errorf("writeTimeToFile: %v", err)
+	}
+
+	return nil
+}
```

Comment:
```suggestion
func writeTimeToFile(chatID string) error {
	path := filepath.Join(cacheDir, chatID)
	content := fmt.Sprintf("%d", time.Now().Add(30*time.Minute).Unix())

	err := os.WriteFile(path, []byte(content), 0644)
	if err != nil {
		fmt.Errorf("writeTimeToFile: %v", err)
	}

	return err
}
```

Reply:
I think using `log.println()` to print error is better than `fmt.Errorf()`?
And according to https://dev.to/mavensingh/strconvitoa-vs-strconvformatint-vs-fmtsprintf-which-is-good-to-convert-int-to-string-in-golang-3iol, `strconv.Itoa` is faster than `fmt.Sprintf()`.

Reply:
Fair enough!